packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -506,13 +501,16 @@ class _RenderCupertinoSwitch extends RenderConstrainedBox {
       visualPosition,
     );
     final double thumbCenterY = offset.dy + size.height / 2.0;
-
-    _thumbPainter.paint(canvas, Rect.fromLTRB(
+    final thumbBounds = Rect.fromLTRB(
```

Comment:
The analyzer is complaining about the missing type annotation.
```suggestion
    final Rect thumbBounds = Rect.fromLTRB(
```

Reply:
Whoops, fixed!